dll/win32/browseui/shellbrowser.cpp

```diff
@@ -306,6 +307,7 @@ class CShellBrowser :
     HACCEL m_hAccel;
     ShellSettings m_settings;
     SBFOLDERSETTINGS m_deffoldersettings;
+    DWORD m_BSF; // BrowserServiceFlags
```

Comment:
`m_BSF` -> `m_BrowserServiceFlags` or `m_BrowserSvcFlags` ... something that is self-documented and doesn't need the comment.

Reply:
But all operations on `m_BSF` use the `BSF_*` flags, should be clear what is stored there. "BrowserServiceFlags" is me guessing what BSF means.